src/main/resources/application.properties

```diff
@@ -1 +1,14 @@
 
+
+spring.h2.console.enabled=true
+spring.datasource.url=jdbc:h2:mem:testdb;NON_KEYWORDS=user,VALUE
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=sa
+spring.datasource.password=
+spring.jpa.hibernate.ddl-auto=update
+spring.jpa.properties.hibernate.format_sql=true
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.H2Dialect
+
+###spring.jpa.properties.javax.persistence.schema-generation.database.action=drop-and-create
```

Comment:
It's best practice to remove commented out code / configuration. If you intend to use it, then maybe add TODO with a relevant comment to flag this?